src/game_main.cpp

```diff
@@ -510,17 +535,7 @@ Game::Game(
   : mpWindow(pWindow)
   , mRenderer(pWindow)
   , mResources(effectiveGamePath(commandLineOptions, *pUserProfile))
-  , mIsShareWareVersion([this]() {
```

Comment:
Do things work correctly without this change? I might be missing something, but if the game itself is working fine, then this check should also work fine in the Webassembly version.